db/migrate/20241011122123_add_benefits_claims_id_to_form_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddBenefitsClaimsIdToFormSubmissions < ActiveRecord::Migration[7.1]
+  def change
+    add_column :form_submissions, :benefits_claims_id, :string
```

Comment:
instead use `benefits_claims_uuid`?

Reply:
I thought about this but the Benefits Claims API seems to return an incrementing integer id, not a UUID. The Benefits Intake API returns a UUIDv4.